### PR TITLE
add setUp, setUpBeforeScript, tearDown and tearDownAfterScript functions execution

### PR DESCRIPTION
**Pull request by cmayo**
## 📚 Description

You can create functions with the following names in your test script to perform actions before and after test/script:

`setUp`

If this function exists in the test script it will be executed before each test execution.

`setUpBeforeScript`

If this function exists in the test script it will be executed only one time before all tests execution.

`tearDown`

If this function exists in the test script it will be executed after each test execution.

`tearDownAfterScript`

If this function exists in the test script it will be executed only one time after all tests execution.

## ✅ To-do list

- [ ] Make sure that all the pipeline passes
- [ ] Make sure to update the `CHANGELOG.md` to reflect the new feature or fix

### CODE REVIEW

**Review by Chemaclass — APPROVED** [review]
Thank you! 🤩  This will close https://github.com/TypedDevs/bashunit/issues/47

**Review by fabriziofs — APPROVED** [review]
Great work ✨

File: tests/unit/helpers_test.sh [hunk 1]
```diff
@@ -39,3 +43,15 @@ function test_getFunctionsToRun_fail_when_duplicates() {
...
+function test_dummyFunction_is_executed_with_execute_function_if_exists() {
+  local function_name='dummyFunction'
+
+  assertEquals "dummyFunction executed" "$(executeFunctionIfExists "$function_name")"
+}
+
+function test_no_function_is_executed_with_execute_function_if_exists() {
+  local function_name='notExistingFunction'
+
+  assertEmpty "$(executeFunctionIfExists "$function_name")"
```
[the remark this answers is not in the document]
Comment (cmayo) [reply tests/unit/helpers_test.sh:56]: Yes, you are right. I'm trying to check why I didn't get any error when with this non-existent method

[the remark this answers is not in the document]
Comment (khru) [reply tests/unit/helpers_test.sh]: No, thanks to you, We've just added the assert https://github.com/TypedDevs/bashunit/pull/93 (it needs a refactor, but the first step is there)

**Pull request merged**